apps/frontend/src/shared-research/ResearchOutputList.tsx

```diff
@@ -28,8 +27,10 @@ const ResearchOutputList: React.FC<ResearchOutputListProps> = ({
     filters,
     currentPage,
     pageSize,
+    searchIndex: !searchQuery.length ? INDEX['desc(addedDate)'] : INDEX.primary,
```

Comment:
you should have one place - one function which is going to return proper index based on search query.
Now you have multiple (two?) locations to make such changes. It should always be just one :)

Reply:
It is necessary in this case because the algolia object is initialized in two different places.

https://github.com/yldio/asap-hub/blob/d3d3653cb75b040b9167ae377a7b70ed03cb5ef8/apps/frontend/src/shared-research/state.ts#L108

https://github.com/yldio/asap-hub/blob/d3d3653cb75b040b9167ae377a7b70ed03cb5ef8/apps/frontend/src/shared-research/ResearchOutputList.tsx#L33